src/styles/sass/features/_base.scss

```diff
@@ -0,0 +1,71 @@
+@import '../abstracts/';
+
+.features {
+    width: 100%;
+    background-color: $color-black;
+    color: $color-white;
+    padding-top: 20px;
+    padding-bottom: 20px;
+
+    .features__header {
```

Comment:
W Sass mamy `&` aby nie powtarzać ciągle tych samych nazw tj. `&__header` (https://css-tricks.com/the-sass-ampersand/)